### DISCUSSION TITLE
Trying to push changes into main

### DISCUSSION THREAD

**Discussion by Alwin224** [microsoft/TinyTroupe#49] [Q&A]
Hello Everyone!

This is my very first open source project and I had a few additions that I made. I was wondering how I could push my forked repository to the main upstream branch. I am unable to do it because I am getting a 403 error that I am not approved to. Is there any way to get approval?

Best,
Alwin

**Comment by knqti** [comment]
Check out the Contributing section in the README. Looks like you'll need to agree to their CLA first.
